Redefine a Kotlin configuration (#212) (#214)
hyperledger/iroha-2-docs: src/guide/kotlin-java.md

Snapshot versions match the Git commits.
To get the latest snapshot,
simply visit the [`iroha-java`](https://github.com/hyperledger/iroha-java/tree/iroha2-dev) repository

> on `iroha-2-dev` branch, look at the commit hashes and copy the short hash of the last commit on the main page.

Reviewer: ```suggestion
on `iroha-2-dev` branch and copy the short hash of the last commit on the main page.
```

![](/img/iroha_java_hash.png)

> If it isn't available, check the [list of commits](https://github.com/hyperledger/iroha-java/commits/iroha2-dev) and find one of the previous hashes of a commit[^1].

Reviewer: ```suggestion
You can also check the [commit history](https://github.com/hyperledger/iroha-java/commits/iroha2-dev) and copy the commit hash of one of the previous commits.
```

    }
}
```

> [^1]: the short form, as well

Reviewer: I don't think this is a necessary note

```suggestion
```
